express-spotify-app/app.js

```diff
@@ -0,0 +1,18 @@
+var SpotifyWebApi = require('spotify-web-api-node');
+
+// Remember to paste here your credentials
+var clientId = '61174fc8ca5c403a84ecbb20d47b131a',
+    clientSecret = '91f38e31ccd24dc18820b515bfcf28d1';
+
+var spotifyApi = new SpotifyWebApi({
+  clientId : clientId,
+  clientSecret : clientSecret
+});
+
+// Retrieve an access token.
+spotifyApi.clientCredentialsGrant()
```

Comment:
This piece of code is asynchronous, so you can have problems if the response is not ready when the app needs it. You can try to wrap it in a function with a callback.